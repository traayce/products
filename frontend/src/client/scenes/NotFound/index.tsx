import * as React from 'react';
import { RouteComponentProps } from 'react-router-dom';

const NotFound: React.SFC<RouteComponentProps<{}>> = () => {
  return (
    <div>
      <h1>404</h1>
      <p>The page you're looking for cannot be found</p>
    </div>
  );
};

export default NotFound;
