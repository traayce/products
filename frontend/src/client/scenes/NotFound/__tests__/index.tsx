import * as React from 'react';
import { MemoryRouter } from 'react-router';
import * as renderer from 'react-test-renderer';
import NotFound from '../index';

describe('NotFound scene', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
