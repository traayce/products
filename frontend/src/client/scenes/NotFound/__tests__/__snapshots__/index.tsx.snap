// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound scene should render correctly 1`] = `
<div>
  <h1>
    404
  </h1>
  <p>
    The page you're looking for cannot be found
  </p>
</div>
`;
